SwiftWisdom/Core/Geometry/Geometry.swift

```diff
@@ -75,78 +75,63 @@ public func *(lhs: CGSize, rhs: CGFloat) -> CGSize {
     return CGSize(width: lhs.width * rhs, height: lhs.height * rhs)
 }
 
-public func random(range: Range<Int>) -> Int {
-    guard let first = range.first, let last = range.last else { return 0 }
-    assert(range.first >= 0)
+public func random(_ range: CountableClosedRange<Int>) -> Int {
+    let first = range.lowerBound
+    let last = range.upperBound
+
+    assert(first >= 0)
```

Comment:
i just realized that this code was being duplicated. We have implemented random(inRange:) somewhere else. ghastly. That other one doesnt have the assertion and is also tested. removing.